### ISSUE TITLE
Preview links not being reported in Broken link report

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12180]
## Description
While I was working on another task, I discovered multiple pages with links pointing to preview links.  These links are being flagged in the CMS, however they were not reported in the broken link report and they had been present for at least a year (probably longer). After speaking with Dave C, we should investigate why these types of links were not bring reported to that we can bring them to editors attention to be resolved.

### Example pages

- https://www.va.gov/birmingham-health-care/programs/current-pharmacy-residents/
- https://www.va.gov/birmingham-health-care/programs/former-pharmacy-residents/
- https://www.va.gov/birmingham-health-care/programs/how-to-apply-for-a-residency/
- https://www.va.gov/birmingham-health-care/programs/pgy2-rotations/
- https://www.va.gov/birmingham-health-care/programs/pharmacy-pgy1-program/
- https://www.va.gov/birmingham-health-care/programs/pharmacy-pgy1-rotations/
- https://www.va.gov/birmingham-health-care/work-with-us/internships-and-fellowships/pharmacy-pgy2-program/
- https://www.va.gov/birmingham-health-care/programs/pharmacy-preceptors/
- https://www.va.gov/birmingham-health-care/programs/why-choose-the-birmingham-va-medical-center/
- https://www.va.gov/birmingham-health-care/programs/current-pharmacy-residents/
- https://www.va.gov/birmingham-health-care/programs/former-pharmacy-residents/
- https://www.va.gov/birmingham-health-care/programs/how-to-apply-for-a-residency/
- https://www.va.gov/birmingham-health-care/programs/pgy2-rotations/
- https://www.va.gov/birmingham-health-care/programs/pharmacy-pgy1-program/
- https://www.va.gov/birmingham-health-care/programs/pharmacy-pgy1-rotations/
- https://www.va.gov/birmingham-health-care/work-with-us/internships-and-fellowships/pharmacy-pgy2-program/
- https://www.va.gov/birmingham-health-care/programs/pharmacy-preceptors/
- https://www.va.gov/birmingham-health-care/programs/why-choose-the-birmingham-va-medical-center/

## Acceptance Criteria
- [ ] Confirm preview links are not reported in broken link report
- [ ] Determine if there is a reason for why they are not
- [ ] Determine if the report should/can be updated to include these links

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by ndouglas** [comment]
I did a quick check and found only 18 -- which I think might be the same ones you list above:

```
./build/vagovdev/birmingham-health-care/policies/index.html
./build/vagovdev/birmingham-health-care/programs/current-pharmacy-residents/index.html
./build/vagovdev/birmingham-health-care/programs/former-pharmacy-residents/index.html
./build/vagovdev/birmingham-health-care/programs/how-to-apply-for-a-residency/index.html
./build/vagovdev/birmingham-health-care/programs/pgy2-rotations/index.html
./build/vagovdev/birmingham-health-care/programs/pharmacy-pgy1-program/index.html
./build/vagovdev/birmingham-health-care/programs/pharmacy-pgy1-rotations/index.html
./build/vagovdev/birmingham-health-care/programs/pharmacy-preceptors/index.html
./build/vagovdev/birmingham-health-care/programs/why-choose-the-birmingham-va-medical-center/index.html
./build/vagovdev/birmingham-health-care/work-with-us/internships-and-fellowships/pharmacy-pgy2-program/index.html
./build/vagovdev/coronavirus-veteran-frequently-asked-questions-esp/index.html
./build/vagovdev/disability/parent-dic-rates/past-rates-2019/index.html
./build/vagovdev/disability/parent-dic-rates/past-rates-2020/index.html
./build/vagovdev/disability/parent-dic-rates/past-rates-2021/index.html
./build/vagovdev/education/other-va-education-benefits/veap/index.html
./build/vagovdev/illiana-health-care/work-with-us/internships-and-fellowships/index.html
./build/vagovdev/initiatives/emergency-room-911-or-urgent-care/index.html
./build/vagovdev/minneapolis-health-care/health-services/suicide-prevention/index.html
```

**Comment by laflannery** [comment]
Your list has more - I was only in the Birmingham section, you have a couple other VAMCs and some Benefits Hub pages

**Comment by ndouglas** [comment]
oh, I see now.  I think you listed 'em twice.

**Comment by laflannery** [comment]
Yep, oops! removed the duplicates so it's just the 9 original in my description

**Comment by laflannery** [comment]
I just found this somewhat related issue because it referenced the Broken link report: #5345.  I wanted to link the 2 because if work will be done or anything maybe we want to look at this other one at the same time

**Comment by stefaniefgray** [comment]
Tagging myself in this ticket so I can keep an eye on it 👀 

This tends to happen around once every 1-2 months or so and seems frequent enough to be worth bumping to medium priority as a possible quick win

cc: @laflannery

**Comment by ndouglas** [comment]
@laflannery @stefaniefgray @TroyA6, I opened a PR that would fix this (at least moving forward) by blocking any attempt to link to a `*.vfs.va.gov` domain.  I'm not sure offhand if we have any links to subdomains of `vfs.va.gov`, though, e.g. `tugboat.vfs.va.gov` or anything like that.  I don't expect that we would, but thought I'd check before suggesting it as a Really Serious Solution™.

**Comment by ndouglas** [comment]
... or I could've searched and found in seconds that we do 😵‍💫

**Comment by stefaniefgray** [comment]
@ndouglas Thank you so much for looking into this! You rock 🎸
